cfme/tests/control/test_alerts.py

```diff
@@ -216,8 +214,8 @@ def test_alert_vm_turned_on_more_than_twice_in_past_15_minutes(
     provider.refresh_provider_relationships()
 
     # preparing events to listen to
-    builder = EventBuilder(get_or_create_current_appliance())
-    base_evt = partial(builder.new_event, target_type='VmOrTemplate', target_name=vm_name)
+    listener = appliance.event_listener()
+    base_evt = partial(listener.new_event, target_type='VmOrTemplate', target_name=vm_name)
```

Comment:
I'm confused what happens here
```listener = appliance.event_listener()``` but doesn't this also ```== register_event``` as per the fixture definition below?